addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -88,6 +88,61 @@ var KanbanRenderer = BasicRenderer.extend({
         start_quick_create: '_onStartQuickCreate',
         quick_create_column_updated: '_onQuickCreateColumnUpdated',
     }),
+    events:_.extend({}, BasicRenderer.prototype.events || {}, {
+        'keydown .o_kanban_record' : '_onRecordKeyDown'
+    }),
+    _focusOnNextCard: function(currentCardElement) {
+        var nextCard = currentCardElement.nextElementSibling;
+        if (nextCard) {
+            nextCard.focus();
+        }
+    },
+    _focusOnPrevousCard: function(currentCardElement) {
```

Comment:
private function, should be documented, should be in private section. Also, Previous, not Prevous